DNN Platform/Components/ClientDependency/Source/ClientDependency.Core.csproj

```diff
@@ -29,7 +29,7 @@
     <WarningLevel>4</WarningLevel>
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
     <DocumentationFile>bin\ClientDependency.Core.XML</DocumentationFile>
-    <NoWarn>1591,1573</NoWarn>
+    <NoWarn>1591,1573,618</NoWarn>

```

Comment:
I don't think we should be ignoring obsolete warnings. Any obsolete code should have a recommended replacement, right? If there is non I think we should keep the warnings

Reply:
These warnings are comes from ClientDependency.Core project, and it contains several obsoleted warnings, if not ignore this maybe we can't reach the goal "less than 10 warnings.".

Reply:
So these are actually coming from the 3rd party code we copied to DNN, right?

Reply:
Yep.